README.md

```diff
@@ -0,0 +1,49 @@
+# global-workflow
+Global Superstructure/Workflow currently supporting the Finite-Volume on a Cubed-Sphere Global Forecast System (FV3GFS)
+
+The global-workflow depends on the following prerequisities to be available on the system:
+
+* workload management platform / scheduler - LSF or SLURM
+* workflow manager - ROCOTO (https://github.com/christopherwharrop/rocoto)
+* modules - NCEPLIBS (various), esmf v8.0.0bs48, hdf5, intel/ips v18, impi v18, wgrib2, netcdf v4.7.0, hpss, gempak (see module files under /modulefiles for additional details)
+* manage_externals - A utility from ESMCI to checkout external dependencies. Manage_externals can be obtained at the following address and should be in the users PATH: https://github.com/ESMCI/manage_externals
+
+The global-workflow current supports the following machines:
+
+* WCOSS-Dell
+* WCOSS-Cray
+* Hera
+
+## Build global-workflow:
+
+### 1. Check out components
+
+The global-workflow uses the manage_externals utility to handle checking out its components. The manic-v1.1.8 manage_externals tag is supported.
+
+Run manage_externals (checkout_externals) while at top of clone:
+
+```
+$ checkout_externals
```

Comment:
I suggest replacing:
```$ checkout_externals```
with
```$ checkout_externals -e Externals.cfg```

This makes it explicit that is being checked out and from where.

Reply:
I like it, I'll make the change, thanks!